app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
+    # ...
+  end
```

Comment:
```suggestion
  def home; end
```
ko có code j thì viết trên 1 line, đưa line comment vào làm j